packages/components/src/lib/SongCard.tsx

```diff
@@ -1,225 +1,221 @@
   };
 
-  const handleTitleClick = () => {
-    onTitleClick?.();
+  const handlePriceClick = (event: MouseEvent | KeyboardEvent) => {
+    event.preventDefault();
+    event.stopPropagation();
+    onPriceClick?.();
   };
 
-  const handleSubtitleClick = () => {
+  const handleSubtitleClick = (event: MouseEvent | KeyboardEvent) => {
+    event.preventDefault();
+    event.stopPropagation();
     onSubtitleClick?.();
   };
 
   const handleKeyPress = useCallback(
-    (action?: () => void) => (event: KeyboardEvent<HTMLDivElement>) => {
-      if ((event.key === "Enter" || event.key === " ") && action) {
-        action();
-      }
-    },
+    (action?: (e: MouseEvent | KeyboardEvent) => void) =>
+      (event: KeyboardEvent) => {
+        if ((event.key === "Enter" || event.key === " ") && action) {
+          action(event);
+        }
+      },
     []
   );
 
   const commonPriceStyles = {
     alignSelf: "start",
     background: "rgba(0, 0, 0, 0.4)",
     borderRadius: "6px",
-    gridArea: "1 / 1 / 2 / 2",
     justifySelf: "end",
     margin: [0.5, 1],
     px: 1,
     py: 0.5,
   };
 
   if (isLoading) {
-    return <SongCardSkeleton />;
+    return (
+      <SongCardSkeleton
+        isSubtitleVisible={ !!subtitle }
+        isTitleVisible={ !!title }
+      />
+    );
   }
 
   return (
-    <Stack
+    <Clickable
       sx={ {
-        maxWidth: ["150px", "150px", `${songCardSize[size]}px`],
-        rowGap: 0.5,
+        display: "flex",
+        justifyContent: "flex-start",
+        position: "relative",
+        width: "100%",
       } }
+      onClick={ onCardClick ? handleCardClick : undefined }
     >
-      <Stack alignItems="center" display="grid" justifyItems="center">
-        <img
-          alt="Song cover art"
-          height={ isWidthAboveMd ? songCardSize[size] : 150 }
-          src={
-            coverArtUrl
-              ? resizeCloudinaryImage(coverArtUrl, {
-                  height: songCardSize[size],
-                  width: songCardSize[size],
-                })
-              : getImageSrc(bgImage)
-          }
-          style={ {
-            borderRadius: "4px",
-            gridArea: "1 / 1 / 2 / 2",
-            objectFit: "cover",
-          } }
-          width={ isWidthAboveMd ? songCardSize[size] : 150 }
-        />
-        { isPlayable && (
-          <IconButton
-            aria-label={ `${isPlaying ? "Stop" : "Play"} song` }
-            sx={ {
-              backgroundColor: "rgba(0, 0, 0, 0.3)",
-              color: theme.colors.white,
-              gridArea: "1 / 1 / 2 / 2",
+      <Stack sx={ { rowGap: 0.5 } } width="100%">
+        <Stack alignItems="center" justifyItems="center" position="relative">
+          <Box
+            aria-label="Song cover art"
+            component="img"
+            height="100%"
+            src={
+              coverArtUrl
+                ? resizeCloudinaryImage(coverArtUrl, {
+                    height: 400,
+                    width: 400,
```

Comment:
Should this be adjusted based on the sizing or 400px covers the basis for the small and large sizes?